docs/api/baseplate/clients/hvac.rst

```diff
@@ -0,0 +1,47 @@
+``baseplate.clients.hvac``
+==========================
+
+.. automodule:: baseplate.clients.hvac
+
+To integrate HVAC with your application, add the appropriate client declaration
```

Comment:
In the same line as my Cass feedback, would it be worth intro'ing Vault and/or HVAC? When I read this first sentence from the perspective of someone who doesn't know Vault or its ecosystem, I am left to dig elsewhere in the docs to figure out what HVAC is/means.